Assets/Scripts/Waypoint Scripts/UpdateWaypoints.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+namespace ISAACS
+{
+    public class UpdateWaypoints : MonoBehaviour
+    {
+
+        public Material selectedPassedLine;
```

Comment:
please add comments to everything!

Reply:
@apollovision

Reply:
We will work on the waypoint scripts in a new pull request.